clients/src/test/java/org/apache/kafka/common/metrics/stats/FrequenciesTest.java

```diff
@@ -89,8 +88,17 @@ public void testBooleanFrequencies() {
         }
         assertEquals(0.25, falseMetric.stat().measure(config, time.milliseconds()), DELTA);
         assertEquals(0.75, trueMetric.stat().measure(config, time.milliseconds()), DELTA);
+    }
 
-        // Record 2 more windows worth of values
+    @Test
+    public void testBooleanFrequenciesStrategy2() {
```

Comment:
Does `testBooleanFrequenciesStrategy2` cover anything different from `testBooleanFrequenciesStrategy1`?

Reply:
Tried just to keep all test scenarios as is to limit the scope of PR. So didn't judge it deeply. Should we?